packages/client-http/src/client/ConfidenceClient.ts

```diff
@@ -17,7 +16,20 @@ type ResolveResponse = {
   resolvedFlags: ResolvedFlag[];
   resolveToken: string;
 };
+type ConfidenceBaseTypes = { boolSchema: {} } | { doubleSchema: {} } | { intSchema: {} } | { stringSchema: {} };
+type ConfidenceFlagSchema = {
+  schema: {
+    [key: string]: ConfidenceBaseTypes | { structSchema: ConfidenceFlagSchema };
+  };
+};
 
+export type ResolvedFlag<T = any> = {
+  flag: string;
+  variant: string;
+  value?: T;
+  flagSchema?: ConfidenceFlagSchema;
```

Comment:
if you resolve a flag without any context then the response will not have a value or a schema. But will have "" for variant and a no segment match reason. So the reason is because it matches the possible return values from the backend.